.golangci.yaml

```diff
@@ -1,66 +1,48 @@
 linters:
   disable-all: true
-  # Enable specific linter
-  # https://golangci-lint.run/usage/linters/#enabled-by-default-linters
   enable:
-    # Default linters
```

Comment:
What's the motivation behind dropping the linters? I copied this from other projects this team owns

Reply:
Some of them were deprecated/archived. I got the latest list from the CAPI project upstream

Reply:
The diff is a bit strange due to the ordering of the list

Reply:
These linter changes are removing
```
- contextcheck
- cyclop
- dupl
- errname
- errorlint
- exhaustive
- forcetypeassert
- funlen
- gochecknoglobals
- gocognit
- gocyclo
- goerr113
- goheader
- ifshort
- makezero
- nestif
- nilnil
- nlreturn
- tagliatelle
- tenv
- wastedassign
- wrapcheck
- wsl
```
and adding
```
- asasalint
- containedctx
- dupword
- errchkjson
- ginkgolinter
- nosprintfhostport
- rowserrcheck
- thelper
- usestdlibvars
```
I would appreciate if we can keep
```
- contextcheck
- goerr113
- wastedassign
- wrapcheck
- wsl
```
To be honest I'm surprised a few of those aren't already in the CAPI list